Makefile

```
@@ -52,6 +59,62 @@ $(services): target/spec $(openapi-generator-jar)
 	mv $(output)/$(models)/$@ $(models)/$@
 	mv $(output)/$(models)/JSON.java $(models)/$@
 
+# Full service + models automation
+bigServices:=balanceplatform checkout storedValue payments payout management legalentitymanagement transfers
+singleFileServices:=balancecontrol binlookup dataprotection storedvalue posterminalmanagement recurring
+
+all: $(bigServices) $(singleFileServices)
+
+$(bigServices): target/spec $(openapi-generator-jar)
+	rm -rf $(models)/$@ $(output)
+	rm -rf src/main/java/com/adyen/service/$@ $(output)
+	$(openapi-generator-cli) generate \
+		-i target/spec/json/$(spec).json \
+		-g $(generator) \
+		-t templates \
+		-o $(output) \
+		--reserved-words-mappings configuration=configuration \
+		--ignore-file-override ./.openapi-generator-ignore \
+		--skip-validate-spec \
+		--model-package $(subst /,.,com.adyen.model.$@) \
+		--library $(library) \
+		--api-package com.adyen.service.$@ \
+        --api-name-suffix Service \
```

I kind like the `Api` suffix, it produces classes like `PaymentsApi` and `ModificationsApi`. Checkout as a whole is a service and these classes are parts of the whole, but not a service on their own. They also form the main API offered by the library to use our REST API's, seems like a good choice of word. ;) 

```suggestion
        --api-name-suffix Api \
```